.pytool/CISettings.py

```diff
@@ -46,8 +45,6 @@ def __init__(self):
 
     def AddCommandLineOptions(self, parserObj):
         group = parserObj.add_mutually_exclusive_group()

```

Comment:
is the group still needed.  I don't see any other parameter using it and i don't know what happens if you create a mutually exclusive group but then never add any arguments.

Reply:
I'll remove it.